### PR TITLE
#1540. Tests error expectations updated

### PR DESCRIPTION
**Pull request by sgrekhov**

### CODE REVIEW

File: Language/Classes/Setters/static_setter_t02.dart [hunk 1]
```diff
@@ -22,6 +22,7 @@ class C {
   static set foo(String s) {
 //           ^
 // [analyzer] unspecified
+    // [cfe] unspecified
```
Comment (johnniwinther) [inline Language/Classes/Setters/static_setter_t02.dart:25]: The indentation is inconsistent here.

**Review by johnniwinther — APPROVED** [review]

**Comment by eernstg** [comment]
Hi, @sgrekhov! Could you add just a few words about the nature of these changes? I'm not quite sure what to look for when I'm reviewing.

**Comment by sgrekhov** [comment]
> Hi, @sgrekhov! Could you add just a few words about the nature of these changes? I'm not quite sure what to look for when I'm reviewing.

This PR fixes https://github.com/dart-lang/co19/issues/1540. Change https://dart-review.googlesource.com/c/sdk/+/268946 was landed, CFE chander reporting an errors so I'm changing the tests accordingly

**Review by eernstg — APPROVED** [review]
I do not understand why the code changed from having try/catch to not having it. However, the test was always expecting a compile-time error, so there's no need for a try/catch construct (unless there is some subtle effect associated with control flow analysis, but I can't see how that could be the case in these tests). So it's fine that the try/catch is gone, but it would be nice to hear just a couple of words about this topic. ;-)

The next thing as that, apparently, the CFE used to report an error on call sites for a name-clashed declaration, and that's no longer the case.

So, LGTM!

**Comment by sgrekhov** [comment]
These `try/catch` preserved here since the times of Dart 1.0. AFAIR  when Dart was not strongly typed, the code which produced a static errors in the analyser, could be run on VM and produced a runtime error there. So, these `try/catch` were for VM meaning "runtime error is Ok here"

**Comment by eernstg** [comment]
OK, thanks!

**Pull request merged**